h2o-core/src/main/java/water/jdbc/SQLManager.java

```diff
@@ -234,16 +233,19 @@ private static void initializeDatabaseDriver(String databaseType) {
     }
   }
 
-  private static class SqlTableToH2OFrame extends MRTask<SqlTableToH2OFrame> {
+  static class SqlTableToH2OFrame extends MRTask<SqlTableToH2OFrame> {
     final String _url, _table, _user, _password, _columns;
     final int _numCol, _nChunks;
     final boolean _needFetchClause;
     final Job _job;
+    final int _cloudSize;
+    int _nthreads;
```

Comment:
I would make this transient and initialize only in setupLocal not both in constructor and setupLocal